expression/builtin_cast_vec.go

```diff
@@ -577,11 +577,49 @@ func (b *builtinCastTimeAsTimeSig) vecEvalTime(input *chunk.Chunk, result *chunk
 }
 
 func (b *builtinCastTimeAsStringSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinCastTimeAsStringSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	var buf *chunk.Column
+	var err error
+	if b.args[0].GetType().Tp == mysql.TypeTimestamp {
```

Comment:
Why do we check this?
The same buf will be returned for ETTimestamp and ETDatetime.

Reply:
ok, got it~